framework/src/task_manager/task_manager_unicast.c

```diff
@@ -136,9 +134,7 @@ int task_manager_reply_unicast(tm_unicast_msg_t *reply_msg)
 		return TM_OUT_OF_MEMORY;
 	}
 	data->msg_size = reply_msg->msg_size;
-	memcpy(data->msg, reply_msg->msg, reply_msg->msg_size);
-
-	handle = taskmgr_get_handle_by_pid(getpid());
+	memcpy(data->msg, reply_msg->msg, reply_msg->msg_size + 1);
```

Comment:
Please don't make a decision by task manager itself. User gives msg_size. Don't do "+ 1" even if it is wrong.

Reply:
If someone gives rigth value, buffer size 10 including null and user gives 10 as size, at that case this causes invalid memory access.